src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -186,6 +186,58 @@ TEST_CASE("CheckCopyMovingNoisyLinearTest", "[FeedForwardNetworkTest]")
 
   // Check whether move constructor is working or not.
   CheckMoveFunction<>(model2, input, output, 1);
+ * Check whether copying and moving network with dropout is working or not.
+ */
```

Comment:
Looks like the merge did something wrong here. Looks like the comment has to be corrected.

Reply:
I have fixed them. Thank you for letting me know!